app/helpers/form_helper.rb

```diff
@@ -0,0 +1,10 @@
+module FormHelper
+  def bool_to_int bool
```

Comment:
Initially I was thinking of adding a `.to_i` method to the `TrueClass` and `FalseClass` but decided to not pollute the ruby classes.